src/zc/intid/tests/test_zope_zcml.py

```diff
@@ -79,13 +80,7 @@ class I(zope.interface.Interface):
         self._check_only_zc_interface()
 
 
-
 def test_suite():
     return unittest.TestSuite([
         unittest.makeSuite(TestZopeIntidZcml),
     ])
-
-test_suite() # coverage
-
-if __name__ == '__main__': # pragma: no cover
-    unittest.main()
```

Comment:
I vote for removing these bits from the other test files too.

Reply:
Okay.